modules/authentication/proc-enabling-authentication-with-microsoft-azure.adoc

```diff
@@ -0,0 +1,147 @@
+[id="enabling-authentication-with-microsoft-azure"]
+= Enabling authentication with Microsoft Azure
+
+{product} includes a Microsoft Azure authentication provider that can authenticate users by using OAuth.
+
+.Prerequisites
+. You have the permission to register an application in Microsoft Azure.
+. You link:https://docs.redhat.com/en/documentation/red_hat_developer_hub/{product-version}/html-single/administration_guide_for_red_hat_developer_hub/index#assembly-add-custom-app-file-openshift_admin-rhdh[added a custom {product-short} application configuration].
+
+.Procedure
+. To allow {product-short} to authenticate with Microsoft Azure, link:https://learn.microsoft.com/en-us/entra/identity-platform/scenario-web-app-sign-user-app-registration?tabs=aspnetcore#register-an-app-by-using-the-azure-portal[create an OAuth Application in Microsoft Azure].
+
+.. In the Azure portal go to link:https://portal.azure.com/#view/Microsoft_AAD_RegisteredApps/ApplicationsListBlade[*App registrations*], create a **New registration** with the configuration:
++
+**Name**:: The application name in Azure, such as __<My {product-short}>__.
+
+.. On the *Home > App registrations > __<My {product-short}>__ > Manage > Authentication* page, *Add a platform*, with the configuration:
+
+*Redirect URI*:: Enter the backend authentication URI set in {product-short}: `pass:c,a,q[{my-product-url}/api/auth/microsoft/handler/frame]`
```

Comment:
I would suggest converting these into substeps rather than using a description list.

Reply:
This is one step: this is in one single configuration screen.